### DISCUSSION TITLE
w5500 internal protocol used

### DISCUSSION THREAD

**Discussion by skylin008** [micropython/micropython#13122] [STM32 / Pyboard]
Hello every one! With stm32 pboard , how to rebuild the firmware used internal udp or tcp protocol with ethernet chip of w5500 , but not used the lwip protocol instead!  Thanks!

**Comment by andrewleech** [comment]
If `#define MICROPY_PY_LWIP    (0)` is set then the stack in the wiznet chip should be used.
